### PR TITLE
Part3 To Do list: interactive list

### PR DESCRIPTION
**Pull request by lucas-crodrigues**
For this part of the project, I was required to do the checkbox and clear completed functions for the to-do list.

The project requirements were:

> ### Project requirements
> 
> - Add a new JavaScript file and import it as a module:    
>     - it will contain methods related to the status updates (`completed`: `true` / `false`).
> - Add event listener to the checkbox (`change`).
> - Update items object's value for `completed` key upon user actions.
> - Implement a function for the "Clear all completed" button (use `filter()` method).
> - Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

Thank you

### CODE REVIEW

File: src/modules/functions.js [hunk 1]
```diff
@@ -52,10 +52,33 @@ const updateIds = () => {
...
+const checkBox = (e) => {
+  tasks = getLocalStorage();
+  const task = e.target;
+  const index = Number(task.id.split('_')[1]);
+
+  if (tasks[index - 1].completed === false) {
+    tasks[index - 1].completed = true;
+  } else if (tasks[index - 1].completed === true) {
+    tasks[index - 1].completed = false;
+  }
```
Comment (cynthiainga) [inline src/modules/functions.js:64]: - [ ] Please the task object's values for completed are updated upon user actions but for you when I check the box the completed value remains `false`  it's supposed to change to `true` when the box is checking :upside_down_face: :upside_down_face:, please kindly fix that :grin: :grin:

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @lucas-crodrigues, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] All linters are passing :heavy_check_mark: 
- [ ] Descriptive pull request :heavy_check_mark: 
- [ ] Good commit history :heavy_check_mark: 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review for more details :point_down: :point_down: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/functions.js (hunk 1 above)
Comment (lucas-crodrigues) [reply src/modules/functions.js:64]: Hi @cynthiainga, thank you for the feedback, I've recorded a loom video showing the function, hopefully, it clears this misunderstanding, thank you very much 😃 
Here's the [loom video](https://www.loom.com/share/274ff09a4d8c4cd4966d79ee154d6d14).

Comment (cynthiainga) [reply src/modules/functions.js:64]: Hi @lucas-crodrigues, I understand what's wrong. But I think it should be better when you refresh the browser the localStorage update automatically. You can create a function to update the storage :grin: :grin:

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @lucas-crodrigues , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**